### ISSUE TITLE
Any EMS in combination with an exterior window screen throws a severe error - should extend the EMS actuator to include this and other options

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#7586]
Issue overview
--------------
When EMS is setting up actuators for Window Shading Control, if the file has any WindowShadingControls that apply shading type ExteriorScreen, these severe errors will be thrown:
```
   ** Severe  ** Missing shade or blind layer in window construction name = 'WINDOW_WITH_SCREEN', surface name = 'PERIMETER_TOP_ZN_3_WALL_NORTH_WINDOW'.
   **   ~~~   ** ...'Control Status' or 'Slat Angle' EMS Actuator cannot be set for a construction that does not have a shade or a blind layer.
   **   ~~~   ** ...Add shade or blind layer to this construction in order to be able to apply EMS Actuator.
```

1. This happens if the idf has *any* EMS, even if there are no actuators directed at a window with an exterior screen. The error is thrown in `EMSManager::SetupWindowShadingControlActuators`.

2a. One option is to simply skip over windows that have an exterior screen and leave things as-is.

2b. A better option would be to set up the actuator for windows with an exterior screen. Looking at `SolarShading::WindowShadingManager` it appears that an EMS actuator for exterior screen would work just fine. Just need to update the documentation to list the `SurfaceWindow(n).ShadingFlag` control value: "4: exterior screen is on".

3. There are additonal values for `SurfaceWindow(n).ShadingFlag` in the code that are not listed in the EMS documentation. It appears that these would work just fine with an EMS actuator, so add this list to the EMS documentation:
```
        //  10: window has an interior shade that is off but may be
        //       triggered on later to control daylight glare
        //  20: window has switchable glazing that is unswitched but may be switched later
        //       to control daylight glare or daylight illuminance
        //  30: window has exterior shade that is off but may be triggered on later
        //       to control daylight glare
        //  60: window has an interior blind that is off but may be
        //       triggered on later to control daylight glare
        //  70: window has an exterior blind that is off but may be
        //       triggered on later to control daylight glare
        //  80: window has a between-glass shade that is off but may be
        //       triggered on later to control daylight glare
        //  90: window has a between-glass blind that is off but may be
        //       triggered on later to control daylight glare
```

### Workaround
These errors can be ignored.

### Details
Some additional details for this issue (if relevant):
 - Version of EnergyPlus 9.2.0
 - PNNL e-mail, Oct 18, 2019

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added to EnergyPlusDevSupport\DefectFiles
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by rraustad** [comment]
Option 2a - it seems if this option is used, then shading controls will not work for windows with exterior screens. What if a shade or blind is applied to between-glass or interior of the window? Will the EMS shading control actuator work for this case? If not, then Option 2b is preferred.

**Comment by mjwitte** [comment]
@rraustad The shading model can't handle both exterior screen plus another shading layer at the same time, so not sure if that's a big loss.

**Comment by lgu1234** [comment]
@rraustad I tried to build a test file with Option2b, not successful. If windows with exterior screens are used, "Window Shading Control" can not be applied. Therefore, Option 2a may be applicable way to eliminate these warnings.

**Comment by mjwitte** [comment]
@lgu1234 But ExteriorScreen is a valid choice for Shading Type in WindowShadingControl. If that doesn't work, we should be fixing that. It's used that way in example file WindowTests.idf.

**Comment by lgu1234** [comment]
@mjwitte Yes. The ExteriorScreen is a valid choice of Shading Type in WindowShadingControl. It should work when I looked at the code. However, it is not a valid choice for EMS Window Shading Control. I am going to investigate a possibility to add this capability in response to Option 2b.

Anyway, Option 2a fix is always valid to remove unnecessary warnings.

**Comment by lgu1234** [comment]
@rraustad @mjwitte I created another PR to address Option 2b at Allow ExteriorScreen used in EMS Window Shading Control #7941. ExteriorScreen is allowed as EMS actuator for Window Shading Control. A test file was uploaded. Could you take a look to decide which is better for this issue: Option 2a or Option 2b.

**Comment by Myoldmopar** [comment]
Soooo it looks like 2b is the right path.  2a would be fine to eliminate the warning, but the better solution is to just fix the missing behavior with 2b.  I'm going to close the 2a PR and review the fix with 2b.  If someone feels different then please speak up directly.

**Comment by Myoldmopar** [comment]
Fixed with #7941
